compat/test/browser/suspense-list.test.js

```diff
@@ -0,0 +1,66 @@
+import { setupRerender } from 'preact/test-utils';
+import React, {
+	createElement,
+	render,
+	Component,
+	Suspense,
+	lazy,
+	Fragment
+} from 'preact/compat';
+import { setupScratch, teardown } from '../../../test/_util/helpers';
+
+const h = React.createElement;
+/* eslint-env browser, mocha */
+
+describe('suspense-list', () => {
```

Comment:
If you are looking for some inspiration, here are the [React SuspenseList tests](https://github.com/facebook/react/blob/master/packages/react-reconciler/src/__tests__/ReactSuspenseList-test.internal.js). Thought they may be useful just to see what scenarios they are testing, not how they are doing. In other words, what the names of their tests and do we want some similar ones. But perhaps that can come later :)

Reply:
I have tried to keep them quite close for now(sans the warning tests).
Will add more to have full parity

Reply:
Another interesting case to consider: re-suspending under a SuspenseList: https://twitter.com/dan_abramov/status/1192077102490824707?s=09

Reply:
Thats an excellent test

Reply:
Thanks